src/core/dbus.c

```diff
@@ -1181,6 +1181,38 @@ int bus_verify_manage_units_async(Manager *m, sd_bus_message *call, sd_bus_error
                         error);
 }
 
+int bus_verify_manage_units_async_full(
+                Manager *m,
+                const char *unit_name,
```

Comment:
Hmm, why is Unit* param replaced with split Manager and unit_name?

Reply:
It's OK to re-typedef Unit if unit.h cannot be included, if that's what concerns you

Reply:
Previously I got 'Unit was already loaded' error if I call `bus_verify_manage_units_async_full` after `transient_unit_from_message`, and pass unit `u` parameter. I don't really understand how transient unit different from normal unit, so I guess I could pass unit name instead.